apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -459,6 +480,7 @@ class ReviewTab extends Component {
 
 export const UnconnectedReviewTab = ReviewTab;
 export default connect(state => ({
+  codeReviewEnabled: state.sectionData.section.codeReviewEnabled,
```

Comment:
Nit (we can fix this another time, I feel like we're going to need to get back in here to simplify some of our wild if statements): I feel like this should be named something that indicates that it relates to the section (eg, sectionCodeReviewEnabled) since we do a lot of additional logic checks to see whether to actually enable code review.

Reply:
Do you mean just in the apps code or everywhere?